package.json

```diff
@@ -69,6 +69,7 @@
     "ignore": "^5.2.0",
     "import-in-the-middle": "^1.3.4",
     "ipaddr.js": "^2.0.1",
+    "is-loopback-addr": "^1.0.1",
```

Comment:
Note to team: looks rather straightforward: https://github.com/vasco-santos/is-loopback-addr/blob/main/src/index.ts